tests/scripts/update_manifest_test.py

```
@@ -0,0 +1,143 @@
+            "1.2.3",
+            "--key-id",
+            "test-key",
+            "--signing-key-file",
+            str(self.private_key),
+            "--output",
+            str(output),
+        ]
+
+    def test_generates_verifiable_signed_envelope(self) -> None:
+        output = self.directory / "stable.json"
+        subprocess.run(self.command(output), check=True)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)